doc/community/developer-guide.md

```diff
@@ -19,10 +19,28 @@ myst:
 
 # Developer guide
 
+Before continuing, ensure you have a working [development environment.](https://ploomber-contributing.readthedocs.io/en/latest/contributing/setup.html)
+
 +++
 
 ## Unit testing
 
+### Running tests
+
+Unit tests are executed on each PR; however, you might need to run them locally.
+
+To run all unit tests:
```

Comment:
i believe that with my added tests on the #319 change, running all unit tests locally started to fail due to a `Too many open files` error. i was able to bypass it by temporarily increasing my [ulimit](https://stackoverflow.com/questions/39537731/errno-24-too-many-open-files-but-i-am-not-opening-files)

`ulimit -n 2048`

another environment issue specific only to my mac, but having instructions published here in case other users encounter it might be worth writing since 256 is the default for macs.